### PR TITLE
Fix Van Allen's CEO by making getCardPlayer explicit.

### PR DESCRIPTION
**Pull request by kberg**
h/t @d-little

### CODE REVIEW

**Comment by kberg** [comment]
Fixes #5572

File: src/server/Game.ts [hunk 1]
```diff
@@ -1454,16 +1454,29 @@ export class Game implements Logger {
     return ret;
   }
 
-  public getCardPlayer(name: CardName): Player {
+  /**
+   * Returns the Player holding this card, or throws.
+   */
+  public getCardPlayerOrThrow(name: CardName): Player {
+    const player = this.getCardPlayerOrUndefined(name);
```
Comment (lizthegrey) [inline src/server/Game.ts:1461]: avoid duplicating the code by calling getCardPlayerOrUndefined, doing the undefined check, and throwing if undefined.

Comment (kberg) [reply src/server/Game.ts:1461]: How is that different from what's in the PR? What am I missing?

**Review by kberg — COMMENTED** [review]
Thanks for the drive-by

File: src/server/Game.ts (hunk 1 above)
Comment (lizthegrey) [reply src/server/Game.ts:1461]: I, uh, didn't have my coffee yet, that's exactly what you did. *facepalm*

**Review by lizthegrey — APPROVED** [review]

File: src/server/Game.ts (hunk 1 above)
Comment (kberg) [reply src/server/Game.ts:1461]: Thanks for the extra confirmation! :coffee:  :clinking_glasses:

**Review by d-little — APPROVED** [review]

**Pull request merged**